io/shared/src/test/scala/fs2/io/net/tcp/SocketSuite.scala

```diff
@@ -221,7 +225,7 @@ class SocketSuite extends Fs2IoSuite with SocketSuitePlatform {
         }
     }
 
-    test("read after timed out read not allowed on JVM or Native") {
+    test("read after timed out read not allowed on JVM or Native".ignore) {
```

Comment:
Is this not passing?

Reply:
Ah, the new polling implementation actually allows this behavior. A read can timeout without corrupting the socket because we are only canceling the `select`. The old implementation using the JDK's `AsynchronousSocketChannel` doesn't support cancelable reads.

Technically the test is still relevant to the old implementations (which I guess we aren't testing anymore).

Reply:
Ah nice, that always felt like an annoying restriction. Can we flip this test around to assert this works as expected now then?

Reply:
Cool, good idea!